File: member-service/src/main/java/com/hedvig/integration/botService/dto/UpdateUserContextDTO.java
package com.hedvig.integration.botService.dto;

import lombok.AllArgsConstructor;
import lombok.Value;

@Value
@AllArgsConstructor
public class UpdateUserContextDTO {

  private String memberId;
  private String personalNumber;

  private String firstName;
  private String lastName;

  private String phoneNumber;
  private String email;

  private String street;
  private String city;
  private String zipCode;

  private boolean hasSigned;
}

File: member-service/src/main/java/com/hedvig/integration/botService/dto/UpdateUserContextDTO.kt
package com.hedvig.integration.botService.dto

class UpdateUserContextDTO(
    val memberId: String,
    val personalNumber: String?,
    val firstName: String?,
    val lastName: String?,
    val phoneNumber: String?,
    val email: String?,
    val street: String?,
    val city: String?,
    val zipCode: String?,
    val hasSigned: Boolean?
)
